.github/workflows/github.yml

```diff
@@ -1,4 +1,4 @@
-name: Publish (GitHub)
+name: Build 'OSConfig for MC' (NRP and policy packages)
```

Comment:
This is a publishing worklow, sure it includes building it but the root name should be `Publish ...` something. Maybe `Publish 'OSConfig for MC (NRP and policy packages)` is a better name?

Reply:
It appears to work fine as it is and new name better reflects the actual purpose.

Reply:
As discussed, we'll change prefix to 'Publish' when/if we'll make this workflow also publish the package (e.g. to a release)